Ghidra/Features/Base/src/main/java/ghidra/app/plugin/core/datamgr/actions/DeleteArchiveAction.java

```diff
@@ -78,11 +78,7 @@ public boolean isAddToPopup(ActionContext context) {
 		}
 
 		DataTypeTreeNode node = (DataTypeTreeNode) selectionPaths[0].getLastPathComponent();
-		if (!(node instanceof FileArchiveNode)) {
-			return false;
-		}
 		return node instanceof FileArchiveNode;
-
```

Comment:
This might be a copy/paste error as with CaptureFunctionDataTypesAction.java. I'm not sure what it should have been here though.